disnake/mentions.py

```diff
@@ -116,8 +116,8 @@ def from_message(cls, message: Message) -> Self:
         )
 
     def to_dict(self) -> AllowedMentionsPayload:
-        parse = []
-        data = {}
+        parse: List[str] = []
+        data: Dict[str, Union[List[int], bool, List[str]]] = {}
```

Comment:
Since there is already a type ignore at the end of the method, we could move it here, making the rest of the method actually type safe:
```py
        data: AllowedMentionsPayload = {}  # type: ignore
```

Reply:
[`cae62d5` (#1329)](https://github.com/DisnakeDev/disnake/pull/1329/commits/cae62d58fba8d11f7f45927e5035f05672a9a11c)